pkg/server/apis/types.go

```diff
@@ -60,20 +60,26 @@ type TraitMeta struct {
 
 //used to present trait which is to be attached and, of which parameters are set
 type TraitBody struct {
-	EnvName      string       `json:"env_name"`
-	Name         string       `json:"name"`
-	Flags        []CommonFlag `json:"flags"`
-	WorkloadName string       `json:"workload_name"`
-	AppGroup     string       `json:"app_group,omitempty"`
-	Staging      string       `json:"staging,omitempty"`
+	EnvName       string       `json:"env_name"`
+	Name          string       `json:"name"`
+	Flags         []CommonFlag `json:"flags"`
+	ComponentName string       `json:"component_name"`
+	AppGroup      string       `json:"app_group,omitempty"`
```

Comment:
```suggestion
	AppGroup      string       `json:"appGroup,omitempty"`
```

We MUST align our style, it feels really bad to see we have so much naming problem.